### ISSUE TITLE
Unusual additional step on DatesRange

### ISSUE THREAD

**Issue by imadmk** [arfedulov/semantic-ui-calendar-react#154]
**Describe the bug**
When I want to reselect the dates on DatesRangeInput, it will first clear the input then I can select the start date, so there are 3 steps and maybe can confuse the user? Is this actually a preferred ux flow or a bug?

**To Reproduce**
1. Go to the demo page
2. Try DatesRangeInput multiple times

**Expected behavior**
Usually, (eg. react-dates) the first step is selecting the start date

**Dependencies versions**
Provide version numbers of following packages:

* semantic-ui-react 0.86.0
* semantic-ui-css 2.4.1
* semantic-ui-calendar-react 0.15.0

**Comment by arfedulov** [comment]
It happens intentionaly. I'll see if it' not very hard to change current behavior

**Comment by imadmk** [comment]
So you agree to make it just 2 steps? I will try to help too

**Comment by arfedulov** [comment]
Yes. I think there is no need in third step

**Comment by xesf** [comment]
I agree is not user friendly to have to click a second time to re-select the date.

**Comment by vladi-strilets** [comment]
Hi, any updates on this? 
Thanks

**Comment by pedro-mass** [comment]
☝️ Pretty please?
